src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carPrice = 40;
+  const maxDiscount = 50;
+  const discount = 20;
+  const lessDays = 3;
+  const moreDays = 7;
+
+  if (days >= moreDays) {
+    return days * carPrice - maxDiscount;
```

Comment:
`days * carPrice` is calculated three times. Do it once, and use variable for this.